src/v2/Apps/Order/Routes/Review/index.tsx

```diff
@@ -81,8 +81,8 @@ export class ReviewRoute extends Component<ReviewProps> {
       const orderOrError =
         this.props.order.mode === "BUY"
           ? (await this.submitBuyOrder()).commerceSubmitOrder?.orderOrError
-          : (await this.submitOffer(setupIntentId)).commerceSubmitOrderWithOffer
-              ?.orderOrError
+          : (await this.submitOffer(setupIntentId))
```

Comment:
not 100% sure, but I guess we haven't touched anything here, should stay with the Exchange mutation instead of the new one